CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.14.3 (2019-10-24)
+* Schema: add last function
```

Comment:
"add function `last/x` which fetches the last entry"
wdyt? It's a tiny bit more descriptive.